tensorboard/plugins/scalar/vz_line_chart/vz-chart-helpers.ts

```diff
@@ -41,6 +41,12 @@ export interface Point {
   dataset: Plottable.Dataset;
 }
 
+export interface TooltipColumn {
+  title: string;
+  // This function computes the value for the string.
+  computeFunction: ((p: Point, smoothingEnabled: boolean) => string);
```

Comment:
Consider passing `options: {smoothingEnabled: boolean}`. It's more extensible, but more importantly it gives a name to the boolean parameter.